docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -24,9 +40,15 @@ class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
   private generateChatItems(chat: ChatData): JSX.Element[] {
     return generateChatProps(chat).map(({ itemType, ...props }, index) => {
       const ElementType = this.getElementType(itemType)
+      const newProps = { ...props, text: undefined }
       return (
         <Chat.Item key={`chat-item-${index}`}>
-          <ElementType {...props} />
+          {itemType === ChatItemTypes.message && (
+            <div style={ariaLiveStyle} role="heading" aria-level={4}>
+              {this.getMessagePreviewForScreenReader(props)}
+            </div>
+          )}
+          <ElementType {...newProps} />
```

Comment:
what these `newProps` stand for? :) Name doesn't explain much. I would suggest to do it explicitly, as it is the only place where those are used:

```jsx
 <ElementType {...props} text={undefined}  />
```